core/lib/dal/src/storage_logs_dedup_dal.rs

```diff
@@ -172,28 +172,45 @@ impl StorageLogsDedupDal<'_, '_> {
         .map(|max| max as u64))
     }
 
-    /// Returns the maximum enumeration index assigned in a specific L1 batch.
-    pub async fn max_enumeration_index_for_l1_batch(
+    pub async fn max_enumeration_index_in_l1_batch(
```

Comment:
Nit: I'd document what this new version of the method does. Also, if I understand its semantics correctly ("get the max enumeration index by a certain L1 batch"), IMO, `max_enumeration_index_by_l1_batch` or `*_at_l1_batch` would be a better name.